cob_moveit_bringup/launch/sensor_manager.xml

```diff
@@ -6,8 +6,9 @@
 
   <!-- Params for the octomap monitor -->
   <param name="octomap_frame" type="string" value="odom_combined"/>
-  <param name="octomap_resolution" type="double" value="0.1"/>
-  <param name="max_range" type="double" value="2.0"/>
+  <param name="octomap_resolution" type="double" value="0.05"/>
+  <param name="max_range" type="double" value="5.0"/>
```

Comment:
do not change values that are used by other robots, too
If you need to set them we might need to consider a robot-specific parameter.yaml within `cob_moveit_config` or proper launch arguments